mapviz_plugins/CMakeLists.txt

```diff
@@ -120,6 +157,7 @@ install(TARGETS ${PROJECT_NAME}
   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
 )
 
-install(FILES mapviz_plugins.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(FILES ${PROJECT_NAME}.xml
```

Comment:
I don't have strong feelings either way, but `catkin_lint` complains if the CMake project name is used as a literal string anywhere in the file.